angular/utility.scss

```diff
@@ -0,0 +1,10 @@
+@function createSimplePalette($color) {
```

Comment:
may want to rename this to `createSimpleDarkPalette` to avoid confusion...a light palette would use 500 as main and (700?) as dark.

Reply:
These palettes weren't as useful as I had originally hoped and I didn't even use them in the light theme; I just pushed out a refactor that completely removes these palettes and replaces them with these: 

    $primary: map-get($theme, primary);
    $accent: map-get($theme, accent);
    $warn: $pxb-red;